src/InlineHelp/InlineHelp.js

```diff
@@ -55,11 +70,11 @@ InlineHelp.propTypes = {
     'right',
     'left',
     'bottom-center' */
```

Comment:
Placement values should be updated; or point to `Popover > props > placement` documentation